test/unit/specs/components/wallet/PageTransactions.spec.js

```diff
@@ -1,20 +1,222 @@
 import PageTransactions from "renderer/components/wallet/PageTransactions"
-import lcdClientMock from "renderer/connectors/lcdClientMock.js"
 import { createLocalVue, shallowMount } from "@vue/test-utils"
 
 describe(`PageTransactions`, () => {
   const localVue = createLocalVue()
   localVue.directive(`tooltip`, () => {})
-  const { stakingParameters, txs, candidates } = lcdClientMock.state
+
+  const addresses = [
+    `cosmos15ky9du8a2wlstz6fpx3p4mqpjyrm5ctpesxxn9`,
+    `cosmos1pxdf0lvq5jvl9uxznklgc5gxuwzpdy5ynem546`
+  ]
+  const validatorAddresses = [
+    `cosmosvaladdr15ky9du8a2wlstz6fpx3p4mqpjyrm5ctqzh8yqw`,
+    `cosmosvaladdr15ky9du8a2wlstz6fpx3p4mqpjyrm5ctplpn3au`,
+    `cosmosvaladdr15ky9du8a2wlstz6fpx3p4mqpjyrm5ctgurrg7n`
+  ]
+  const allTransactions = [
```

Comment:
@sabau has made the case that we should put this kind of data in the JSON folder

Reply:
I'm thinking to open a ticket about that because I'm spreading this around all PRs XD

Reply:
ok makes sense. @faboweb do you want to move this to a JSON file or shall we merge as is?

Reply:
I propose merging as is and do the refactor based on Karolys proposal later.